autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/ui/credential/management/viewing/AutofillManagementCredentialsMode.kt

```diff
@@ -176,9 +179,12 @@ class AutofillManagementCredentialsMode : DuckDuckGoFragment(R.layout.fragment_a
 
     private fun launchDeleteLoginConfirmationDialog() {
         this.context?.let {
+            val title = stringBuilder.stringForDeleteAllPasswordsDialogConfirmationDialogTitle(numberToDelete = 1)
```

Comment:
Seems weird to read `stringForDeleteAllPasswords` but then passing as argument `1`. should we create another method like we have for `stringForDeleteLoginsConfirmationDialogMessage` instead?

Reply:
See https://github.com/duckduckgo/Android/pull/4445#discussion_r1583009435